test/interpreter_functional/plugins/kbn_tp_run_pipeline/public/np_ready/app/components/main.tsx

```diff
@@ -61,12 +61,9 @@ class Main extends React.Component<{}, State> {
         data: new DataAdapter(),
       };
       return getExpressions()
-        .execute(expression, {
+        .execute(expression, context || { type: 'null' }, {
```

Comment:
Why is the type the string 'null'?

Reply:
This is to replicate the former behaviour of `ExpressionDataHandler`, see here:

- https://github.com/elastic/kibana/blob/master/src/plugins/expressions/public/execute.ts#L59